lib/client-assets.php

```diff
@@ -127,7 +127,9 @@ function gutenberg_override_translation_file( $file, $handle ) {
 	}
 
 	// Ignore scripts that are not found in the expected `build/` location.
-	$script_path = gutenberg_dir_path() . 'build/' . substr( $handle, 3 ) . '/index.js';
+	$indexFilename = ( defined( 'SCRIPT_DEBUG') && SCRIPT_DEBUG ) ? 'index.js' : 'index.min.js';
+
+	$script_path = gutenberg_dir_path() . 'build/' . substr( $handle, 3 ) . "/$indexFilename";
```

Comment:
Even though I did it as part of this changet-set, now that I read the jsdoc for this function again and given this filters the file for loading translations (AFAIU), I'm not sure if we should change the filename to `index.min.js` 🤔. Could someone shed some light? Is it okay to leave it as is?

Reply:
> I'm not really sure if we should change the filename to `index.min.js` here

Are there places where the exact filename matters? For example, the Gutenberg plugin overrides the Gutenberg assets that are present in Core with the newer ones in the plugin. Does the exact filename play any role there? Or does it involve only symbolic ids of the scripts?

Reply:
I don't know. I'll have a look, but I'd love to get insights from @grzim, @ockham or @youknowriad about that.